Core/GDCore/Extensions/Builtin/AudioExtension.cpp

```diff
@@ -343,6 +343,20 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsAudioExtension(
           "res/actions/music.png")
       .AddCodeOnlyParameter("currentScene", "")
       .MarkAsComplex();
+  extension
+      .AddAction(
+          "FadeVolume",
+          _("Fade the volume of a sound played on a channel."),
+          _("Fade the volume of a sound played on a channel within a given time in seconds."),
+          _("Fade the volume of a sound on channel n°_PARAM1_ to a volume of _PARAM2_ within _PARAM3_ seconds"),
+          _("Sounds on channels"),
+          "res/actions/music24.png",
+          "res/actions/music.png")
+      .AddCodeOnlyParameter("currentScene", "")
+      .AddParameter("expression", _("ID of the channel"))
```

Comment:
Please use the same name as the other parameters referring to a channel identifier :)